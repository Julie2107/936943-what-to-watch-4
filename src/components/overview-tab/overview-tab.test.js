import React from "react";
import renderer from "react-test-renderer";
import OverviewTab from "./overview-tab.jsx";

const movie = {title: `Mindhunter`, poster: `http://placekitten.com/245/175`, cover: `img/bg-the-grand-budapest-hotel.jpg`, genre: `drama`, releaseYear: 2000, src: ``, rating: 5, ratingNumber: 100, ratingValue: ``, starring: [``, `1`, `3`], director: ``, reviews: []};

it(`Render Overview tab`, () => {
  const tree = renderer
    .create(<OverviewTab
      movie = {movie}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
