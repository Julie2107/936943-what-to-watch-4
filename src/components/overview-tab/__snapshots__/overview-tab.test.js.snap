// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Overview tab 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      5
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        
      </span>
      <span
        className="movie-rating__count"
      >
        100
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p />
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        , 1, 3
         and other
      </strong>
    </p>
  </div>,
]
`;
